defaults/main.yml

```diff
@@ -48,6 +48,8 @@ nginx_http_params:
   - server_tokens off
   - types_hash_max_size 2048
 
+nginx_http_params: { nginx_http_default_params }
```

Comment:
I don't think this syntax is good, it break on CI : https://travis-ci.org/jdauphant/ansible-role-nginx/builds/189216001#L1713

It will be more : `nginx_http_params: "{{ nginx_http_default_params }}"`